paasta_tools/utils.py

```diff
@@ -2086,6 +2086,14 @@ def get_deployd_startup_oracle_enabled(self) -> bool:
         """
         return self.config_dict.get("deployd_startup_oracle_enabled", True)
 
+    def get_deployd_max_service_instance_failures(self) -> int:
+        """Determines how many times a service instance entry in deployd's queue
+        can fail before it will be removed from the queue.
+
+        :returns: An integer
+        """
+        return self.config_dict.get("deployd_max_service_instance_failures", 100)
```

Comment:
In practice I never saw anything this high, it as always like 20 for some reason

Reply:
Sure, I'll drop it to that